scout/constants/acmg.py

```diff
@@ -60,22 +60,24 @@
                     (
                         "PS1",
                         {
-                            "short": "Known pathogenic aa (HQ)",
+                            "short": "Known pathogenic aa",
                             "description": "Same amino acid change as a previously established pathogenic variant regardless of nucleotide change",
                         },
                     ),
                     (
                         "PS2",
                         {
-                            "short": "De novo (confirmed)",
+                            "short": "<i>De novo</i> (confirmed)",
                             "description": "De novo (both maternity and paternity confirmed) in a patient with the disease and no family history",
+                            "documentation": 'Strength can be modified based on <a href="https://clinicalgenome.org/site/assets/files/3461/svi_proposal_for_de_novo_criteria_v1_1.pdf" target="_blank">SVI <i>de novo</i></a>.',
                         },
                     ),
                     (
                         "PS3",
                         {
-                            "short": "Functional damage (HQ)",
+                            "short": "Functional damage",
```

Comment:
Here it seems somewhat clear the intention was to indicate the requirement in the criterium that "the evidence is strong". Adding strength modification and more flexible Brnich reference here supersedes this.